### PR TITLE
added hex logo to the package

### PR DESCRIPTION
**Pull request by zatch3301**
let me know if the logo is okay or do you need any changes to it.
logo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
-
 <!-- README.md is generated from README.Rmd. Please edit that file -->
```
Comment (philip-khor) [inline README.md:1]: Please edit README.Rmd and then knit.

**Review by philip-khor — CHANGES_REQUESTED** [review]

**Review by philip-khor — CHANGES_REQUESTED** [review]
overall design looks good to me, perhaps you could try to use a dark shade of blue for the background instead of brown? should not be the exact tone of blue as the Bank's logo to avoid confusion that this is an official package.

image

**Comment by zatch3301** [comment]
logo

**Comment by philip-khor** [comment]
looks good to me, thanks!

**Pull request merged**